doc/src/documentation-style-guide.rst

```diff
@@ -0,0 +1,1278 @@
+coding conventions, creating your environment, picking an issue to fix, and
+opening a pull request, please read our full `Development Workflow
+<https://github.com/sympy/sympy/wiki/Development-workflow>`_ guide.
+
+Narrative Documentation Guidelines
+==================================
+
+Extensive documentation, or documentation that is not centered around an API
+reference, should be written as a narrative document in the Sphinx docs (located
+in the `doc/src directory
+<https://github.com/sympy/sympy/tree/master/doc/src>`_). The narrative documents
+do not reside in the Python source files, but as standalone restructured files
+in the doc/src directory. SymPy’s narrative documentation is defined as the
+collective documents, tutorials, and guides that teach users how to use SymPy.
+Reference documentation should go in the docstrings and be pulled into the RST
+with autodoc. The RST itself should only have narrative style documentation
+that is not a reference for a single specific function.
+
+Best Practices for Writing Documentation
+----------------------------------------
+
+Please follow these formatting, style, and tone preferences when writing
+documentation.
+
+Formatting Preferences
+^^^^^^^^^^^^^^^^^^^^^^
+
+In order for math and code to render correctly on the SymPy website, please
+follow these formatting guidelines.
+
+Math
+~~~~
+
+Text that is surrounded by dollar signs $ _ $ will be rendered as LaTeX math.
+Any text that is meant to appear as LaTeX math should be written as ``$math$``.
+In the HTML version of the docs, MathJax will render the math.
+
+**Example**
+
+::
+
+    The Bessel $J$ function of order $\nu$ is defined to be the function
+    satisfying Bessel’s differential equation.
+
+LaTeX Recommendations
+~~~~~~~~~~~~~~~~~~~~~
+
+* Avoid unnecessary ``\left`` and ``\right`` (but be sure to use them when they
+  are required).
+* Avoid unnecessary ``{}``. (For example, write ``x^2`` instead of ``x^{2}``.)
+* Use whitespace in a way that makes the equation easiest to read.
+
+**Examples**
+
+Correct::
+
+    \int \sin(x)\,dx
+
+Incorrect::
+
+    \int \sin{\left( x\right)}\, dx
+
+Code
+~~~~
+
+Text that should be printed verbatim, such as code, should be surrounded by a
+set of double backticks ``like this``.
+
+**Example**
+
+::
+
+    To use this class, define the ``_rewrite()`` and ``_expand()`` methods.
+
+Sometimes a variable will be the same in both math and code, and can even
+appear in the same paragraph, making it difficult to know if it should be
+formatted as math or code. If the sentence in question is discussing
+mathematics, then LaTeX should be used, but if the sentence is discussing the
+SymPy implementation specifically, then code should be used.
+
+In general, the rule of thumb is to consider if the variable in question were
+something that rendered differently in code and in math. For example, the Greek
+letter α would be written as ``alpha`` in code and $\alpha$ in LaTeX. The
```

Comment:
The LaTeX here should be formatted as code so that it shows as is and not LaTeX rendered.

```suggestion
letter α would be written as ``alpha`` in code and ``$\alpha$`` in LaTeX. The
```

Reply:
Fixed this in the latest commit, thanks!